my_list.rb

```diff
@@ -0,0 +1,27 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*items)
+    @list = items
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
+end
+
+# Example usage
+list = MyList.new(1, 2, 3, 4)
+
+# Test #all?
+puts(list.all? { |e| e < 5 }) # Should output: true
+puts(list.all? { |e| e > 5 }) # Should output: false
+
+# Test #any?
+puts(list.any? { |e| e == 2 }) # Should output: true
+puts(list.any? { |e| e == 5 }) # Should output: false
+
+# Test #filter
+puts(list.filter(&:even?)) # Should output: [2, 4]
```

Comment:
- [ ] As shown in the project requirement below, the output for this method should be `[2,4]`. Please kindly adopt this change to your project :+1: 
- project requirement
![fil](https://github.com/Mike47ip/enumerable/assets/89055502/18173c8e-39ea-439e-91f9-83f1a99cdecc)
- your code
![fil](https://github.com/Mike47ip/enumerable/assets/89055502/75980b3e-f6f2-4e53-a800-75525d59876f)